pkg/model/dev.go

```diff
@@ -128,6 +128,11 @@ type BuildInfo struct {
 	ExportCache      string        `yaml:"export_cache,omitempty"`
 }
 
+// Returns the path to the Dockerfile
+func (b *BuildInfo) GetDockerfilePath() string {
```

Comment:
@mnevadom yes, that would fail. What should we do there @pchico83 @jLopezbarb ? I think we should fail if the path to the Dockerfile has a folder (since that's what the context field is for) but I'm not sure how much of a breaking change that is.  We could also try to de-duplicate the path if needed.

Reply:
I would also fail in that case. Does the error mention the `Dockerfile` path that is trying to load? If yes I think that should be enough to know that you should remove the folder from the `dockerfile` field